src/cordova/android/OutlineAndroidLib/build.gradle

```diff
@@ -1,5 +1,6 @@
 // Top-level build file where you can add configuration options common to all sub-projects/modules.
 plugins {
     // This version number must be kept in sync with AGP_VERSION in cordova-android.
-    id 'com.android.library' version '7.2.1' apply false
+    id 'com.android.library' version '7.4.1' apply false
```

Comment:
This may be breaking our Android build: https://github.com/Jigsaw-Code/outline-client/actions/runs/4133200855/jobs/7149446491